pages/Documents/ConversationalAI/Third-PartyBots/public-api.md

```diff
@@ -878,5 +878,11 @@ Example response of a command returned by the API will look like this with state
 | 200  | OK - request for the given API succeeded.                            |
 | 400  | Bad request - Problem with body or query parameters.                 |
 | 401  | Unauthorized - Invalid bearer token.                                 |
+| 403  | Forbidden - If the request was marked as security risk by Reblaze.   |
 | 404  | Not Found - If the provided conversation Id is invalid or not found. |
 | 500  | Internal server error.                                               |
+
+### Limitations
+
+- To ensure the safety of the API we are using [Reblaze](https://www.reblaze.com/). if the request throws Security Violation Error (403 - Forbidden) reason could be that your request body contains information that caused Reblaze to mark your request positive as a security risk. To understand security concepts of Reblaze please refer to the official documentation [here](https://gb.docs.reblaze.com/product-walkthrough/security/concepts). You can also refer to the information on best practices to avoid the false positives marking of valid request [here](https://gb.docs.reblaze.com/using-the-product/best-practices/dealing-with-false-positive)
```

Comment:
**If** the request throws Security Violation Error (403 - Forbidden) **, the** reason